Tribler/Test/Core/Modules/MetadataStore/tools.py

```diff
@@ -0,0 +1,137 @@
+import filecmp
+import os.path
+import threading
+import time
+from datetime import datetime
+
+from pony.orm import db_session
+
+from Tribler.Core.Modules.MetadataStore.base import start_orm
+from Tribler.pyipv8.ipv8.keyvault.crypto import ECCrypto
+
+# FIXME global vars! shadowing!
+crypto = ECCrypto()
+key = crypto.generate_key('curve25519')
+public_key = key.pub().key_to_bin()
+
+
+def are_dir_trees_equal(dir1, dir2):
+    """
+    Shameless copypaste from Stackexchange.
```

Comment:
Source please.

Reply:
https://stackoverflow.com/questions/4187564/recursive-dircmp-compare-two-directories-to-ensure-they-have-the-same-files-and